test/UnitTests/MSTestAdapter.PlatformServices.UnitTests/Deployment/AssemblyLoadWorkerTests.cs

```diff
@@ -22,7 +22,7 @@ public void GetFullPathToDependentAssembliesShouldReturnV1FrameworkAssembly()
             GetReferencedAssembliesSetter = () => [v1AssemblyName],
         };
 
-        var mockAssemblyUtility = new Mock<IAssemblyUtility>();
+        var mockAssemblyUtility = new Mock<IAssemblyUtility>(MockBehavior.Loose);
```

Comment:
I feel this analyzer is too annoying. There is nothing very wrong with not specifying the "default" `MockBehavior.Loose`.